### DISCUSSION TITLE
Multithreading in CBC

### DISCUSSION THREAD

**Discussion by Masik97** [coin-or/Cbc#636] [Discussion]
Hello
I compiled my version of Cbc using:
./coinbrew fetch Cbc
./coinbrew build Cbc --static --enable-cbc-parallel

Then when I tried ./cbc -threads -4 -unitTest it works and multithreading aslo working
but when i tried ./cbc wedding-16.mps -threads 4 then i have this problem
![image](https://github.com/coin-or/Cbc/assets/69193330/9ac59722-8943-4136-9b3f-59834c04cff4)
I am using Ubuntu

**Comment by jhmgoossens** [comment]
You have to explicitly ask cbc to solve the problem for you, so try to add “-solve” to the command line. The options are taken in order. You can experiment with commands by starting cbc without any options added—the interactive mode. 

See [Documentation](https://github.com/coin-or/Cbc#documentation). 

Does that help?

**Reply by Masik97** [reply]
Yes, it was so easy, but too hard for me. Thank you)

**Comment by jhmgoossens** [comment]
Great!
